src/main/java/com/manardenza/utils/DatabaseSeeder.java

```diff
@@ -1,6 +1,67 @@
 package com.manardenza.utils;
 
+import com.manardenza.entity.Hotel;
+import com.manardenza.entity.Room;
+import com.manardenza.entity.User;
+import lombok.Getter;
+
+import java.util.ArrayList;
+import java.util.List;
+
 public class DatabaseSeeder {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1118)